opm/input/eclipse/Schedule/MSW/WellSegments.cpp

```diff
@@ -62,8 +64,8 @@ namespace Opm {
     }
 
 
-    WellSegments::WellSegments(const DeckKeyword& keyword) {
-        this->loadWELSEGS(keyword);
+    WellSegments::WellSegments(const DeckKeyword& keyword, const UnitSystem& unit_system) {
+        this->loadWELSEGS(keyword, unit_system);
```

Comment:
I don't see a better way right now, but having the `UnitSystem` be a parameter to the constructor makes it seem like the `WellSegments` object might plausibly retain a reference (or a full copy) of the `unit_system` argument.  This is something to keep in mind for future work, and I'm not saying we should do anything about it right now.

Reply:
> object might plausibly retain a reference (or a full copy) 

I thought about that, but the only usage of the UnitSystem is for the purpose of outputting the message from this PR, so I decided to pass it around.

Reply:
I can do that. Let me update it.